ppb/systems/__init__.py

```diff
@@ -22,6 +26,22 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 from ppb.systems.pg import EventPoller as PygameEventPoller  # To not break old imports.
+from ppb.assets import Asset
+import io
+
+
+class Image(Asset):
```

Comment:
Dunno on adding new things to `systems.__init__`. If we're making changes to the renderer and adding new things they should move to its own file.

Reply:
Yeah, I was wondering if we wanted to move the renderer to its own module.

Reply:
It's one of the places we deal with pygame, so probably needs to go to `systems.pg`. (There's even argument for making pg.py into a subpackage of its own, but uh. . . not right now.)